src/app/components/media/MediaExpanded.js

```diff
@@ -141,6 +168,27 @@ class MediaExpandedComponent extends Component {
             authorUserName={authorUsername}
           />
         );
+      // } else if (isPender && timelineEnabled) {
```

Comment:
@amoedoamorim is this code waiting to be uncommented? if not, then we can delete it

Reply:
I'll check with @amoedoamorim on that, that was the initial Player bit

Reply:
fixed